ajax/libs/thrift/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "thrift",
+  "filename": "thrift.min.js",
+  "description": "node.js bindings for the Apache Thrift RPC system",
+  "homepage": "https://thrift.apache.org/",
+  "keywords": [
+    "thrift",
+    "rpc",
+    "framework",
+    "serialization",
+    "cross-language"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://git-wip-us.apache.org/repos/asf/thrift.git"
+  },
+  "version": "0.10.0",
+  "author": {
+    "name": "Apache Thrift Developers",
+    "email": "dev@thrift.apache.org",
+    "url": "https://thrift.apache.org"
+  },
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://git-wip-us.apache.org/repos/asf/thrift.git",
+    "fileMap": [
+      {
+        "basePath": "lib/js/src",
+        "files": [
+          "thrift*"
```

Comment:
@qwe19272375 sorry about that  I was not familiar with nodejs in the past.
I think that using this path is fine.
But the `bathPath` should be `lib`
And the `files` should be `"js/src/thrift*"`

Then every thing looks good to me.
Thank you.